<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>705</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>705</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>990</width>
    <height>705</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>我是你爸爸</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/img/Baocuo1.jpg</normaloff>:/new/prefix1/img/Baocuo1.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 220, 217);
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: rgb(238, 231, 215);
border-radius:20px;</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>971</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>310</y>
     <width>381</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 207, 215);

border-radius:20px;</string>
   </property>
   <item>
    <property name="text">
     <string>2021.1.5 11：34   温度过高，所以报错了！！！！！</string>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021.1.5 11：34   温度过高，所以报错了！！！！！</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021.1.5 11：34   温度过高，所以报错了！！！！！</string>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>12</red>
       <green>0</green>
       <blue>2</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>126</green>
       <blue>30</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021.1.5 11：34   温度过高，所以报错了！！！！！</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021.1.5 11：34   温度过高，所以报错了！！！！！</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021.1.5 11：34   温度过高，所以报错了！！！！！</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>620</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(63, 220, 181);

border-radius:20px;</string>
   </property>
   <property name="text">
    <string>🔨</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>620</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 216, 104);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>🚮</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>620</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 82, 67);

border-radius:20px;</string>
   </property>
   <property name="text">
    <string>❌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>620</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:20px;
background-color:rgb(180, 169, 111);</string>
   </property>
   <property name="text">
    <string>💡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>620</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(166, 175, 255);
border-radius:20px;
</string>
   </property>
   <property name="text">
    <string>🔒</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>温度传感器</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>湿度传感器</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>烟雾传感器</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>可燃气体传感器</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>309</y>
     <width>571</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(34, 207, 255);</string>
         </property>
         <property name="text">
          <string>湿度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 126, 30);
border-radius:20px;
lineout:none;</string>
         </property>
         <property name="text">
          <string>可燃气体</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 82, 67);</string>
         </property>
         <property name="text">
          <string>温度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 216, 104);</string>
         </property>
         <property name="text">
          <string>烟雾</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>listWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>radioButton_5</zorder>
  <zorder>radioButton_6</zorder>
  <zorder>radioButton_7</zorder>
  <zorder>radioButton_8</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_4</zorder>
  <zorder>widget_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
